eras/shelley/test-suite/test/Test/Cardano/Ledger/Shelley/Rewards.hs

```diff
@@ -805,9 +784,6 @@ rewardTests =
       testProperty "provenance does not affect result" (newEpochProp 100 (sameWithOrWithoutProvenance @C testGlobals)),
       testProperty "ProvM preserves Nothing" (newEpochProp 100 (nothingInNothingOut @C)),
       testProperty "ProvM preserves Just" (newEpochProp 100 (justInJustOut @C)),
-      testProperty "oldstyle (aggregate immediately) matches newstyle (late aggregation) with provenance off style" (newEpochProp chainlen (oldEqualsNew @C)),
-      testProperty "oldstyle (aggregate immediately) matches newstyle (late aggregation) with provenance on style" (newEpochProp chainlen (oldEqualsNewOn @C)),
-      testProperty "Reward Provenance works" $
```

Comment:
This test, `rewardsProvenance`, no longer makes sense. The function that the new reward pulsers uses to pulse is not in the provenance monad.